lib/upload_creator.rb

```diff
@@ -427,4 +427,25 @@ def add_metadata!
     @upload.secure = UploadSecurity.new(@upload, @opts).should_be_secure?
   end
 
+  private
+
+  def animated?(file)
```

Comment:
`animated?` is always called with `@file` as an argument, so the arg should probably be removed. Especially since this method makes an assumption that there's just one file (by storing frame count in an instance variable)